packages/components/pages/f-checkout/src/components/Checkout.vue

```diff
@@ -414,9 +404,6 @@ export default {
          */
         async initialise () {
             this.setExperimentValues(this.experiments);
-            this.isLoading = true;
```

Comment:
Does this not need to stay to avoid the big we had? It's set to false in resetLoadingState.